specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2019-05-01/machineLearningServices.json

```diff
@@ -0,0 +1,3261 @@
+        },
+        "passwords": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Password"
+          }
+        }
+      }
+    },
+    "Password": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "readOnly": true,
+          "type": "string"
+        },
+        "value": {
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "PaginatedComputeResourcesList": {
+      "type": "object",
+      "description": "Paginated list of Machine Learning compute objects wrapped in ARM resource envelope.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "An array of Machine Learning compute objects wrapped in ARM resource envelope.",
+          "items": {
+            "$ref": "#/definitions/ComputeResource"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "A continuation link (absolute URI) to the next page of results in the list."
+        }
+      }
+    },
+    "ComputeResource": {
+      "type": "object",
+      "description": "Machine Learning compute object wrapped into ARM resource envelope.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        },
+        {
+          "type": "object",
+          "properties":{
+            "properties": {
+              "description": "Compute properties",
+              "$ref": "#/definitions/Compute"
+            }
+          }
+        }
+      ]
+    },
+    "Compute": {
+      "type": "object",
+      "description": "Machine Learning compute object.",
+      "discriminator": "computeType",
+      "properties": {
+        "computeType": {
+          "description": "The type of compute",
+          "$ref": "#/definitions/ComputeType"
+        },
+        "computeLocation": {
+          "description": "Location for the underlying compute",
+          "type": "string"
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "The provision state of the cluster. Valid values are Unknown, Updating, Provisioning, Succeeded, and Failed.",
+          "enum": [
+            "Unknown",
+            "Updating",
+            "Creating",
+            "Deleting",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          },
+          "readOnly": true
+        },
+        "description": {
+          "type": "string",
+          "description": "The description of the Machine Learning compute."
+        },
+        "createdOn": {
+          "type": "string",
+          "description": "The date and time when the compute was created.",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "modifiedOn": {
+          "type": "string",
+          "description": 
```

Comment:
This is missing the `computeType` discriminator property and is the reason the builds are failing.